stackit/services/postgresflex/user/resource.go

```diff
@@ -327,6 +327,7 @@ func (r *userResource) ImportState(ctx context.Context, req resource.ImportState
 	resp.Diagnostics.Append(resp.State.SetAttribute(ctx, path.Root("instance_id"), idParts[1])...)
 	resp.Diagnostics.Append(resp.State.SetAttribute(ctx, path.Root("user_id"), idParts[2])...)
 	tflog.Info(ctx, "Postgresflex user state imported")
+	tflog.Warn(ctx, "The user password is not imported as it is only available upon creation of a new user. The password field will be empty.")
```

Comment:
Minor nitpick, shouldn't this be before the info log? So that the info log marks the end of the import